tests/unit/test_charm_2.py

```diff
@@ -215,3 +258,103 @@ def in_same_line(content, *matches):
                 ),
                 "db info in config should takes precedence over the db relation"
             )
+
+    def test_wp_install_cmd(self):
+        """
+        act: generate wp-cli command to install WordPress
+        assert: generated command should match current config and status
+        """
+        consensus = self._setup_replica_consensus()
+        install_cmd = self.harness.charm._wp_install_cmd()
+        self.assertIn(
+            "--admin_user=admin",
+            install_cmd,
+            "admin user should be \"admin\" with the default configuration"
+        )
+        self.assertIn(
+            "--admin_password={}".format(consensus["default_admin_password"]),
+            install_cmd,
+            "admin password should be the same as the default_admin_password in peer relation data"
+        )
+        self.harness.update_config({
+            "initial_settings": """\
+            user_name: test_admin_username
+            admin_email: test@test.com
+            admin_password: test_admin_password
+            """
+        })
+        install_cmd = self.harness.charm._wp_install_cmd()
+        self.assertIn("--admin_user=test_admin_username", install_cmd)
+        self.assertIn("--admin_email=test@test.com", install_cmd)
+        self.assertIn("--admin_password=test_admin_password", install_cmd)
+
+    def test_core_reconciliation_before_peer_relation_ready(self):
+        """
+        arrange: before peer relation established
```

Comment:
Any other pre-conditions? E.g., the charm has been created?

Reply:
Updated, thanks!